sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2451,17 +2463,28 @@ object Radar {
      * @see [](https://radar.com/documentation/api#send-a-custom-event)
      *
      * @param[name] The name of the conversion.
-     * @param[data] The metadata associated with the conversion.
+     * @param[metadata] The metadata associated with the conversion.
      * @param[callback] A callback.
      */
     @JvmStatic
-    fun logConversion(name: String, data: JSONObject?, callback: RadarLogConversionCallback) {
+    fun logConversion(name: String, metadata: JSONObject?, callback: RadarLogConversionCallback) {
         if (!initialized) {
             callback.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
 
             return
         }
 
+        // if trackOnce() has been called in the last 60 seconds, don't call it again
```

Comment:
nit: it is not if it had been _called_ in the last 60 seconds, it is that it has been _returned_